index.bs

```diff
@@ -170,6 +170,11 @@ with using |origin|, [=ASCII serialization of an origin|serialized=] and [=isomo
 as the [:Origin:] header of the request.
 When establishing a session, the client MUST NOT provide any [=credentials=].
 
+A [=WebTransport session=] |session| is <dfn for=session>draining</dfn> when the
```

Comment:
The key point here is that the server or the resource indicate that the session should be terminated cleanly as soon as possible.